README.md

```diff
@@ -96,21 +98,30 @@ Add the artifacts under `build/install/xenocp/bin` to your `PATH`.
 
 ### Inputs
 
-XenoCP requires two inputs, defined in a YAML file as [CWL inputs]. E.g., `inputs.yml`:
+XenoCP requires three inputs, defined in a YAML file as [CWL inputs]. E.g., `inputs.yml`:
 
 ```
 bam:
   class: File
   path: sample.bam
 ref_db_prefix: /references/ref.fa
+aligner: "bwa aln"
 ```
 
-`bam` is the input sample BAM 
- and `ref_db_prefix`, the basename of the reference assembly that should be cleansed. 
-For example, a prefix of `MGSCv37.fa` would assume
+`bam` is the input sample BAM.
+ `ref_db_prefix`, the basename of the reference assembly that should be cleansed. 
```

Comment:
Remove extra leading whitespace.
Are you saying that for BWA it's a prefix, but for STAR it's a directory?  If so, I would be explicit.